contrib/seeds/makeseeds.py

```diff
@@ -1,32 +1,118 @@
 #!/usr/bin/env python
 #
-# Generate pnSeed[] from Pieter's DNS seeder
+# Generate seeds.txt from Pieter's DNS seeder
 #
 
-NSEEDS=600
+NSEEDS=512
+
+MAX_SEEDS_PER_ASN=2
+
+MIN_BLOCKS = 337600
+
+# These are hosts that have been observed to be behaving strangely (e.g.
```

Comment:
I guess we have this in someone's TODO list already.
https://github.com/dashpay/dash/blob/v0.12.1.x/src/chainparamsseeds.h#L10